miner/src/client.rs

```diff
@@ -70,11 +74,7 @@ impl Rpc {
                             .await
                             .map(|res|res.into_body())
                         {
-                            Ok(mut body) => body
-                                .data()
-                                .await
-                                .ok_or(RpcError::NoRespData)
-                                .and_then(|res| res.map_err(RpcError::Http)),
+                            Ok(body) => StreamBody::new(body).await,
```

Comment:
I think this should be fixed by [to_bytes](https://docs.rs/hyper/0.14.11/hyper/body/fn.to_bytes.html)